.dockerignore

```diff
@@ -1 +1,2 @@
-build
\ No newline at end of file
+build
+target
```

Comment:
drive-by: don't copy unnecessary build output into the docker image. this makes things slightly better when building docker locally.